modules/virt-understanding-node-maintenance.adoc

```diff
@@ -1,25 +1,23 @@
 // Module included in the following assemblies:
-//
-// * virt/node_maintenance/virt-node-maintenance.adoc
-// * virt/node_maintenance/virt-setting-node-maintenance.adoc
+// virt/node_maintenance/virt-about-node-maintenance.adoc
 
 [id="virt-understanding-node-maintenance_{context}"]
 = Understanding node maintenance mode
 
-Placing a node into maintenance marks the node as unschedulable and drains all 
-the virtual machines and pods from it. Virtual machine instances that have a 
-`LiveMigrate` eviction strategy are live migrated to another node without loss 
-of service. This eviction strategy is configured by default in virtual machine 
-created from common templates but must be configured manually for custom 
-virtual machines. 
+Nodes can be placed into maintenance mode using the `oc adm` utility, or using `NodeMaintenance` custom resources (CRs).
 
-Virtual machine instances without an eviction strategy will be deleted on the 
-node and recreated on another node. 
+Placing a node into maintenance marks the node as unschedulable and drains all the virtual machines and pods from it. Virtual machine instances that have a `LiveMigrate` eviction strategy are live migrated to another node without loss of service. This eviction strategy is configured by default in virtual machine created from common templates but must be configured manually for custom virtual machines.
+
+Virtual machine instances without an eviction strategy are shut down. Virtual machines with a `RunStrategy` of `Running` or `RerunOnFailure` are recreated on another node. Virtual machines with a `RunStrategy` of `Manual` are not automatically restarted.
 
 [IMPORTANT]
 ====
-Virtual machines must have a persistent volume claim (PVC) with a shared 
-ReadWriteMany (RWX) access mode to be live migrated.
+Virtual machines must have a persistent volume claim (PVC) with a shared ReadWriteMany (RWX) access mode to be live migrated.
 ====
 
+When installed as part of OpenShift Virtualization, Node Maintenance Operator watches for new or deleted `NodeMaintenance` CRs. When a new `NodeMaintenance` CR is detected, no new workloads are scheduled and the node is cordoned off from the rest of the cluster. All pods that can be evicted are evicted from the node. When a `NodeMaintenance` CR is deleted, the node that is referenced in the CR is made available for new workloads.
```

Comment:
`ReadWriteMany`

Reply:
I don't understand the comment @ahardin :) - can you provide more detail please?

Reply:
Sure! Looks like further up in the file, there's an instance of ReadWriteMany that should be `ReadWriteMany`. Not exactly sure why my comment is so far down, but that's what I meant :smile: